include/SDL_events.h

```diff
@@ -87,6 +87,8 @@ typedef enum
 
     SDL_LOCALECHANGED,  /**< The user's locale preferences have changed. */
 
+    SDL_POLLSENTINEL, /**< Signals the end of an event poll cycle */
```

Comment:
This would probably have to be somewhere out of the way, like right before user events (0x7FFF).